content/departments/engineering/dev/tools/grpc_cs_enablement.md

```diff
@@ -0,0 +1,110 @@
+# gRPC CS Enablement Guide
+
+# **What is gRPC?**
+
+[gRPC](https://grpc.io/) is a high-performance [RPC](https://en.wikipedia.org/wiki/Remote_procedure_call) framework developed by Google.
```

Comment:
Do folks in CS know what an RPC framework is? Or should we add one sentence to explain what it is?